diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSmokeTest.kt

```diff
@@ -195,6 +196,17 @@ class DiktatSmokeTest : FixTestBase("test/smoke/src/main/kotlin",
         Assertions.assertTrue(unfixedLintErrors.isEmpty())
     }
 
+    @Test
+    @Tag("DiktatRuleSetProvider")
+    fun `disable charters`() {
+        val rulesConfigPascalCaseEnum: List<RulesConfig> = listOf(
+            RulesConfig(DIKTAT_COMMON, true,
+                mapOf("disableChapters" to "1,coments,3,4,5,6"))  // mistake made on purpose
+        )
+        fixAndCompare("Example1Expected.kt", "Example1Test.kt", rulesConfigPascalCaseEnum)
+        unfixedLintErrors.assertEquals()
```

Comment:
Are `unfixedLintErrors` empty? It would be better if there would be some errors from second chapter (it should be active, since in `disableChapters` it is by the incorrect name?), so that we can ensure everything works as intended.